drivers/counter/counter_ll_stm32_rtc.c

```diff
@@ -197,17 +197,12 @@ static u32_t rtc_stm32_get_top_value(struct device *dev)
 }
 
 
-static int rtc_stm32_set_top_value(struct device *dev, u32_t ticks,
-				counter_top_callback_t callback,
-				void *user_data)
+static int rtc_stm32_set_top_value(struct device *dev,
+				   const struct counter_top_cfg *cfg)
 {
 	const struct counter_config_info *info = dev->config->config_info;
 
-	ARG_UNUSED(dev);
```

Comment:
This parameter is still unused in this function.

Reply:
used here: `const struct counter_config_info *info = dev->config->config_info;`